contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -172,7 +172,7 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 
     /* Inside todo-list component */
     template: {
-      v-on:increased-count="updateCount"
+      on:increased-count="updateCount"
```

Comment:
Hmm my bad, i meant the `data` object for render functions https://vuejs.org/v2/guide/render-function.html#v-model

Perhaps you could just change it to `template: '...<item v-on:increasedCount="updateCount" v-for="item in items"/>...'`